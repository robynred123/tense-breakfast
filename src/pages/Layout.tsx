import { Outlet } from 'react-router-dom';
import { Header } from '../components/Header';

export const Layout = (props: any) => {
  return (
    <>
      <Header />
      <Outlet />
    </>
  );
};
