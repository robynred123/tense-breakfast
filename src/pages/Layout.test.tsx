import renderer from 'react-test-renderer';
import { Layout } from './Layout';

describe('<Layout />', () => {
  it('Should render the layout component successfully', () => {
    // Struggling to render chldren in test
    const view = renderer.create(<Layout>Hi</Layout>);
    // eslint-disable-next-line testing-library/await-async-query
    const image = view.root.findByType('img');
    expect(view.toJSON()).toMatchSnapshot();
    expect(image.props.src).toEqual('spill-logo.png');
  });
});
