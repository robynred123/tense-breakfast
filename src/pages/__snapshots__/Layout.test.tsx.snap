// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> Should render the layout component successfully 1`] = `
<div
  className="MuiBox-root css-1b8r6wj"
>
  <div
    className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-lg-4 css-1iu26gx-MuiGrid-root"
    >
      <img
        alt="spill logo"
        src="spill-logo.png"
        style={
          Object {
            "minWidth": "200px",
            "width": "50%",
          }
        }
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-lg-8 css-vok43g-MuiGrid-root"
    />
  </div>
</div>
`;
